Index: .env/.env.development
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>APP_HOST=0.0.0.0\r\nAPP_PORT=8000\r\n\r\nJSON_LOGS=true\r\nGUNICORN_WORKERS=1\r\nPROJECT_NAME=PersonalApi\r\n\r\nDB_HOST=localhost\r\nDB_PORT=5432\r\nDB_NAME=my_api\r\nDB_USER=root\r\nDB_PASSWORD=1234\r\n\r\n\r\n\r\nBACKEND_CORS_ORIGINS=[\"*\"]\r\nSECRET_KEY=\r\nSENDINBLUE_API_KEY=\r\n\r\nENVIRONMENT=development\r\nFRONTEND_URL=\r\n\r\nDB_POOL_SIZE=500\r\nDB_MAX_OVERFLOW=100
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env/.env.development b/.env/.env.development
--- a/.env/.env.development	(revision 140cc27b5fdb9aa745be6e3003d9f54cdfd2ac74)
+++ b/.env/.env.development	(date 1735755287013)
@@ -1,24 +1,17 @@
 APP_HOST=0.0.0.0
-APP_PORT=8000
-
+APP_PORT=8001
 JSON_LOGS=true
 GUNICORN_WORKERS=1
 PROJECT_NAME=PersonalApi
 
-DB_HOST=localhost
-DB_PORT=5432
-DB_NAME=my_api
-DB_USER=root
-DB_PASSWORD=1234
-
-
+POSTGRES_HOST=localhost
+POSTGRES_USER=root
+POSTGRES_PASSWORD=1234
+POSTGRES_DB=my_api
 
 BACKEND_CORS_ORIGINS=["*"]
-SECRET_KEY=
-SENDINBLUE_API_KEY=
 
 ENVIRONMENT=development
-FRONTEND_URL=
 
 DB_POOL_SIZE=500
-DB_MAX_OVERFLOW=100
\ No newline at end of file
+DB_MAX_OVERFLOW=100
Index: rough_page.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/rough_page.py b/rough_page.py
--- a/rough_page.py	(revision 140cc27b5fdb9aa745be6e3003d9f54cdfd2ac74)
+++ b/rough_page.py	(date 1735755692957)
@@ -6,3 +6,18 @@
 
 
 
+import os
+
+from config import Config
+
+# Mock environment variables before loading the config
+os.environ["ENVIRONMENT"] = "development"
+os.environ["PROJECT_NAME"] = "PersonalApi"
+os.environ["POSTGRES_HOST"] = "localhost"
+os.environ["POSTGRES_USER"] = "root"
+os.environ["POSTGRES_PASSWORD"] = "1234"
+os.environ["POSTGRES_DB"] = "my_api"
+os.environ["DB_POOL_SIZE"] = "500"
+os.environ["DB_MAX_OVERFLOW"] = "100"
+
+config = Config()  # Now load the config
Index: .idea/personal_api.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"PYTHON_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <excludeFolder url=\"file://$MODULE_DIR$/.venv\" />\r\n    </content>\r\n    <orderEntry type=\"jdk\" jdkName=\"Python 3.12 (personal_api)\" jdkType=\"Python SDK\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/personal_api.iml b/.idea/personal_api.iml
--- a/.idea/personal_api.iml	(revision 140cc27b5fdb9aa745be6e3003d9f54cdfd2ac74)
+++ b/.idea/personal_api.iml	(date 1735754195893)
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/.venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.12 (personal_api)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.13 (personal_api)" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
 </module>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"Black\">\r\n    <option name=\"sdkName\" value=\"Python 3.12 (personal_api)\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 140cc27b5fdb9aa745be6e3003d9f54cdfd2ac74)
+++ b/.idea/misc.xml	(date 1735754195960)
@@ -3,4 +3,5 @@
   <component name="Black">
     <option name="sdkName" value="Python 3.12 (personal_api)" />
   </component>
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.13 (personal_api)" project-jdk-type="Python SDK" />
 </project>
\ No newline at end of file
